packages/cryptoplease_ui/lib/src/status_widget.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:flutter/material.dart';
+
+class CpStatusWidget extends StatelessWidget {
+  const CpStatusWidget({
+    Key? key,
+    required this.child,
```

Comment:
It should provide a stricter interface, e.g. optional `Widget title` and `Widget content`, that would apply the corresponding text styles automatically.